### PR TITLE
fix: block-test unregister warning

### LINKED ISSUE #770: block-test triggers warning about extra unregister
**Category**

**Component**
block-test

**Describe the bug**
Warning about extra unregister call is triggered when this plugin is loaded.
```
No extension mapping for name "test_many_blocks_mutator" found to unregister
```

**To Reproduce**

1. Start the block-test plugin by running `npm run start` from the plugins/block-test directory.
2. Observe warning

**Expected behavior**

No warning.

### PR DESCRIPTION
**Pull request by melanie-t**
Fixes #770 

Changes made
- Add `isRegistered` check to `block-test` plugin before calling unregister

### CODE REVIEW

File: plugins/block-test/src/mutators.js [hunk 1]
```diff
@@ -108,7 +108,9 @@ const MANY_BLOCKS_MUTATOR = {
   },
 };
 
-Blockly.Extensions.unregister('test_many_blocks_mutator');
+if (Blockly.Extensions.isRegistered('test_many_blocks_mutator')) {
+  Blockly.Extensions.unregister('test_many_blocks_mutator');
+};
```
Comment (alschmiedt) [inline plugins/block-test/src/mutators.js]: Can you remove this extra semicolon?

**Review by alschmiedt — COMMENTED** [review]
One small change and then should be all set : )

File: plugins/block-test/src/mutators.js (hunk 1 above)
Comment (melanie-t) [reply plugins/block-test/src/mutators.js]: Done, thank you!

**Review by alschmiedt — APPROVED** [review]
Looks good to me(LGTM), thank you for your work on this!

**Pull request merged**